CMakeLists.txt

```diff
@@ -329,9 +330,18 @@ else()
   set(VIAMCPPSDK_GRPCXX_INCLUDE_DIRS ${GRPCXX_INCLUDE_DIRS})
   set(VIAMCPPSDK_GRPCXX_LIBRARY_DIRS ${GRPCXX_LIBRARY_DIRS})
   set(VIAMCPPSDK_GRPCXX_LIBRARIES ${GRPCXX_LIBRARIES})
+  
+  # We need to set the reflection library, which unfortunately does not 
+  # have a `.pc` file. To do so, we duplicate the `VIAMCPPSDK_GRPCXX_LIBRARIES` 
+  # value, assume its first element is the base dependency for GRPC, and 
+  # prepend our new list with a new value, equal to that base dependency plus 
```

Comment:
It isn't easy to see here, but these four lines have trailing whitespace. Personally, I like to configure my editor to make trailing whitespace visible so I don't forget to clean it up.